### ISSUE TITLE
ChatInterface appears vertically compressed in Safari

### ISSUE THREAD

**Issue by Gauntlet173** [gradio-app/gradio#7908]
### Describe the bug

When loading a ChatInterface in Safari, the area of the chatbot is vertically very small, limited to being able to display two one-line message bubbles before scrolling. The same interface loading on Edge and Chrome has the expected size. May be related to #7714, but happens regardless of whether the ChatInterface element is contained.

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```python
import gradio as gr
def echo(message,history):
  return message
chat = gr.ChatInterface(echo)
chat.launch()
```

### System Info

```shell
It's happening on a colleagues machine, I don't have the spec.
```

### Severity

I can work around it

**Comment by dawoodkhan82** [comment]
@Gauntlet173 I can't seem to reproduce this issue in safari. Can you double check this is still an issue?

**Comment by msanand** [comment]
+1
I'm seeing this issue on Safari on Mac as well. Refer screenshot. The conversation is restricted to a very small space on top, and does not fill the height available.
(gradio v4.26.0, gradio-client v0.15.1)

<img width="1503" alt="image" src="https://github.com/gradio-app/gradio/assets/1041954/5ee6271c-3070-470a-a362-9894d73aac9c">

**Comment by keenranger** [comment]
+1
I also have same issue. This not only affects macOS safari, but iOS safari and iOS chrome which internally use safari.

**Comment by ssuzuki-github** [comment]
I have discovered that this phenomenon does not occur in version 4.16.0, but it started occurring in version 4.17.0.

**Comment by abidlabs** [comment]
Closed via #9571
